fdbrpc/Locality.cpp

```diff
@@ -141,6 +153,8 @@ ProcessClass::Fitness ProcessClass::machineClassFitness( ClusterRole role ) cons
 			return ProcessClass::OkayFit;
 		case ProcessClass::LogRouterClass:
 			return ProcessClass::OkayFit;
+		case ProcessClass::NoAssignClass:
+			return ProcessClass::OkayFit;
```

Comment:
It feels weird to me that something called "NoAssign" can have things assigned to it?

Reply:
In the same spirit as this, I think I'd prefer we actually rename this from NoAssign to something more explicit, like Coordinator or CoordinatorOnly, etc. That way it may be more clear to someone not as well versed in the nuances of process classes that this process not only can run coordinators, but will be preferred by coordinators auto for running coordinators.

Reply:
I will rename this class CoordinatorClass, and make it give NeverAssign fit for the ClusterController role.